packages/mimir/example/pubspec.yaml

```diff
@@ -12,7 +12,8 @@ dependencies:
   http: ^1.0.0
 
   # You can replace this with the current version of mimir
-  mimir: ^0.0.2+1
```

Comment:
would you mind reverting this how it was before? I use `melos` to manage this

Reply:
This has been reverted, curious as to why 'path: ../' is not used though, as your example gets out of sync?

Reply:
`melos` updates version constraints in all packages when you create a new release. If you develop with melos, it'll create a dependency_overrides.yaml (or whatever it is called) that will override the version constraint for you to point to the local package.